src/overlays/actors/ovl_En_Ik/z_en_ik.h

```diff
@@ -2,16 +2,45 @@
 #define Z_EN_IK_H
 
 #include "global.h"
+#include "objects/object_ik/object_ik.h"
 
 struct EnIk;
 
 typedef void (*EnIkActionFunc)(struct EnIk*, GlobalContext*);
 
+#define IK_PARAMS_01 1
+#define IK_PARAMS_FF 0xFF
+
+typedef struct {
+    /* 0x00 */ Gfx* unk_00;
+    /* 0x04 */ Vec3f unk_04;
+    /* 0x10 */ Vec3f unk_10;
+    /* 0x1C */ Vec3s unk_1C;
+    /* 0x24 */ s32 unk_24;
+} EnIkStruct; // size = 0x28
```

Comment:
Looks like this is an effect struct, unk_24 appears to be to do with whether to actually draw it, seems to be used as boolean? The vectors are probably position/velocity/rotation from how they're used?